README.md

```diff
@@ -1,7 +1,7 @@
 # git-utils
 A collection of useful git tools. Currently the only tool here is a git command to sync an upstream repository with your local copy and your fork.
 
-> :warning: **If you are not on Linux**: these scripts have only been tested on Linux so far. Please open an issue if you find any compatibility problems.
+> :warning: **If you are not on Linux/MacOS**: these scripts have only been tested on Linux and MacOS so far. Please open an issue if you find any compatibility problems.
```

Comment:
I don't think we should use `usr/local/bin` because that is shared between users. `~/bin` is local to your user. I'll document what to do in this situation in a resolution to https://github.com/Cali0707/git-utils/issues/16